src/components/interfaces/HMI_API.xml

```diff
@@ -4535,7 +4556,7 @@
   <function name="ClosePopUp" messagetype="response">
     <description>Provides the result of operation.</description>
   </function>
-  <function name="OnResetTimeout" messagetype="notification">
+  <function name="OnResetTimeout" messagetype="notification" deprecated="true" since="5.0">
```

Comment:
@LitvinenkoIra are you sure that it is correct notation? shouldn't it be until='5.0' ?

Reply:
@LuxoftAKutsan I don't see any "until" fields in API.
and these changes I take from original proposal
https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md